config/initializers/ensure_semester_and_settings_present.rb

```diff
@@ -1,5 +1,6 @@
 # Makes sure at least one semester and one settings model is in the system
 # before the app starts.
+# TODO: This should not run on setup
```

Comment:
Can you maybe file this as an issue instead?

Reply:
Added issue !